src/components/radioButton/RadioButton.js

```diff
@@ -165,21 +191,45 @@ class RadioButton extends BaseComponent {
     return style;
   }
 
+  getIconStyle() {
+    const {iconStyle} = this.getThemeProps();
+    const style = [this.styles.image];
+
+    return [style, iconStyle];
+  }
+
+  renderLabel() {
+    return (
+      this.props.label && (<Text marginL-10 style={this.props.labelStyle} >{this.props.label}</Text>)
```

Comment:
Destruct the props you're using at the beginning of the method. 
Other than readability,  it will also save you from writing `this.props.label` twice

Reply:
@M-i-k-e-l you missed this one (: